src/Select.js

```diff
@@ -205,43 +195,47 @@ export class Select extends Component {
         const menu = React.cloneElement(this.props.menu, menuProps)
 
         return (
-            <div
-                className={className}
-                ref={this.selectRef}
-                onFocus={this.onFocus}
-                onKeyDown={this.onKeyDown}
-            >
-                <InputWrapper
-                    onToggle={this.onToggle}
-                    inputRef={this.inputRef}
-                    tabIndex={tabIndex}
-                    error={error}
-                    warning={warning}
-                    valid={valid}
-                    disabled={disabled}
-                    dense={dense}
-                >
-                    {input}
-                </InputWrapper>
+            <React.Fragment>
                 {open && (
-                    <MenuWrapper
-                        maxHeight={maxHeight}
-                        selectRef={this.selectRef}
-                        menuRef={this.menuRef}
-                        menuTop={menuTop}
-                        menuLeft={menuLeft}
-                        menuWidth={menuWidth}
-                    >
-                        {menu}
-                    </MenuWrapper>
+                    <ScreenCover onClick={this.onOutsideClick} transparent />
```

Comment:
Why not rendering the Menu inside the ScreenCover, one less condition in the code? At least that's the intended usage of the ScreenCover.

I can see though how this works this way quite well.. so not a recommendation but rather a question

Reply:
If you render it inside the `ScreenCover` the `ScreenCover` tries to align it in the middle of the screen by putting the `children` in a `div` with these styles:

```
div {
    position: absolute;
    top: 50%;
    left: 50%;
    width: auto;
    height: auto;
    transform: translate(-50%, -50%);
}
```

Reply:
Oh, I forgot